### PR TITLE
Change all uses of "rotational velocity" to "angular velocity"

### PR DESCRIPTION
**Pull request by tom-overton**
In #1319, someone noted that "angular velocity" is the correct term for what I was calling "rotational velocity". However, the whole codebase was using "rotational velocity", so I said I'd stick with it for that PR and then fix it once the PR got merged. I then proceeded to forget about that until just now.

Besides changing "rotational velocity" to "angular velocity", this PR also cleans up some rotation names in BgNumaHana, since they were a bit weird.

### CODE REVIEW

File: src/overlays/actors/ovl_En_Pp/z_en_pp.c [hunk 1]
```diff
@@ -556,14 +556,14 @@ void EnPp_Idle(EnPp* this, PlayState* play) {
...
-            if ((this->maskBounceRotationalVelocity < 0x64) &&
+            if ((this->maskBounceAngularVelocity < 0x64) &&
                 (fabsf(this->actor.world.rot.y - this->targetRotY) < 100.0f)) {
                 Math_ApproachF(&this->actor.speed, 1.0f, 0.3f, 1.0f);
             }
 
-            Math_SmoothStepToS(&this->actor.world.rot.y, this->targetRotY, 1,
-                               (this->maskBounceRotationalVelocity + 0x258), 0);
-            Math_SmoothStepToS(&this->maskBounceRotationalVelocity, 0, 1, 0x1F4, 0);
+            Math_SmoothStepToS(&this->actor.world.rot.y, this->targetRotY, 1, (this->maskBounceAngularVelocity + 0x258),
```
Comment (engineer124) [inline src/overlays/actors/ovl_En_Pp/z_en_pp.c:564]:
```suggestion
            Math_SmoothStepToS(&this->actor.world.rot.y, this->targetRotY, 1, this->maskBounceAngularVelocity + 0x258,
```

File: src/overlays/actors/ovl_En_Pp/z_en_pp.c [hunk 2]
```diff
@@ -602,11 +602,10 @@ void EnPp_Charge(EnPp* this, PlayState* play) {
             this->targetPos.z += distanceFromWorldPos.z;
         }
 
-        Math_SmoothStepToS(&this->actor.world.rot.y, this->targetRotY, 1, (this->maskBounceRotationalVelocity + 0x7D0),
-                           0);
+        Math_SmoothStepToS(&this->actor.world.rot.y, this->targetRotY, 1, (this->maskBounceAngularVelocity + 0x7D0), 0);
```
Comment (engineer124) [inline src/overlays/actors/ovl_En_Pp/z_en_pp.c:605]:
```suggestion
        Math_SmoothStepToS(&this->actor.world.rot.y, this->targetRotY, 1, this->maskBounceAngularVelocity + 0x7D0, 0);
```

File: src/overlays/actors/ovl_Obj_Tokeidai/z_obj_tokeidai.c [hunk 3]
```diff
@@ -377,15 +377,15 @@ void ObjTokeidai_TowerClock_SlideOff(ObjTokeidai* this, PlayState* play) {
 
     if (this->aerialClockFaceSpeed < 0x80) {
         thisx->shape.rot.x = this->slidingClockFaceAngle - 0x4000;
-        this->fallingClockFaceRotationalVelocity = 0x28;
+        this->fallingclockFaceAngularVelocity = 0x28;
```
Comment (engineer124) [inline src/overlays/actors/ovl_Obj_Tokeidai/z_obj_tokeidai.c:380]: `fallingclock` -> `fallingClock` I assume this wasn't intentional?

**Review by engineer124 — COMMENTED** [review]
Thanks for this!

File: src/overlays/actors/ovl_Obj_Tokeidai/z_obj_tokeidai.c (hunk 3 above)
Comment (tom-overton) [reply src/overlays/actors/ovl_Obj_Tokeidai/z_obj_tokeidai.c:380]: Whoops, let me fix this one. It's supposed to be how it was before.

**Review by engineer124 — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Review by isghj5 — APPROVED** [review]

**Pull request merged**